packages/lesswrong/components/posts/PostSubmit.tsx

```diff
@@ -60,10 +61,7 @@ export type PostSubmitProps = FormButtonProps & {
   classes: ClassesType
 }
 
-const requestFeedbackKarmaLevel = forumSelect({
-  EAForum: 200,
-  default: 100,
-})
+const requestFeedbackKarmaLevel = requestFeedbackKarmaLevelSetting.get()
```

Comment:
I think we should move this inside the component otherwise we don't get the ability to update the value without a server restart that we usually get with database settings.